### ISSUE TITLE
Generate a version metadata file which can be used by dotnet-install

### ISSUE THREAD

**Issue by mmitche** [dotnet/core-setup#8779]
When the runtime or other components generates a stabilized build, it publishes to a non-stable directory, with an identical set of stable and suffixed asset names inside. This is so that dotnet-install, which makes an assumption that the version to install will match the asset name, will work properly.  Example:

```
3.0.100-preview3-123545\sdk-3.0.100-preview3-123545.zip (contents are 3.0.100 branded) 
3.0.100-preview3-123545\sdk-3.0.100.zip (contents are 3.0.100 branded) 
```

We publish both so that we can push the final branded artifacts on release day, and the suffixed assets will work with dotnet-install. dotnet-install, however, will install the sdk, then look for the installed version (3.0.100-12345) rather than 3.0.100 because it knows nothing about the contents of the zip. It will fail.

We want to solve two problems

..tbc

**Comment by dagood** [comment]
@mmitche What will this be used for? Is it required for servicing 3.1?

**Comment by mmitche** [comment]
No, primarily about getting a working dotnet-install when are building stable bits.  Does not need to be done before 3.1 ships.
